src/styles/block/catalog.scss

```diff
@@ -0,0 +1,11 @@
+.catalog {
+  @include grid-colums-count;
+
+  display: grid;
+  grid-template-columns: repeat($grid-colums-1, $card-width);
```

Comment:
There is a typo in the variable name '$grid-colums-1'. It should follow the code style rules for correct spelling. Please correct it to '$grid-columns-1' if that is the intended name.